app/configuration.rb

```diff
@@ -27,8 +27,9 @@ module GitlabMonitor
     #Implementation of class responsible for showing notifications in system. Specify hide timeout [s].
     #You can use equations like 60*5 -> 5 minutes
     config.notifier          = (RbConfig::CONFIG['host_os'].match(/mswin|windows/i) \
-                                  ? WindowsNotificationExecutor 
+                                  ? WindowsNotificationExecutor
                                   : LinuxNotificationExecutor).new(time: 5)
+    config.link_enabled        = true
```

Comment:
Tak wiem. Myślałem o tym długo i mocno i zrobiłem ostatecznie tak bo:
* chciałem zachować prostotę konfiguracji
* te stałe i tak były globalne
* to zdaje się nie była jedyna stała używana w tylko jednym miejscu
* hash z opcjami był spójny dla obu wykonawców, a z tym byłaby nadmiarowa dana dla windowsa (co więcej nieużywana i myląca bo tagi wycina niezależnie od niej)